src/UnitTests/Analysis/IndirectCallRewriterTests.cs

```diff
@@ -287,5 +318,19 @@ public void Icrw_TwoArgumentsNoFuncs()
                 "Fragments/icrw/indirect_call_two_arguments.asm",
                 "Analysis/IcrwTwoArgumentsNoFuncs.txt");
         }
+
+        [Test]
+        public void Icrw_TypeReferenceToFunc()
+        {
+            var a = m.Reg32("a");
+            var b = m.Reg32("b");
+            var fn = m.Reg32("fn");
+            fn.DataType = RefFnPtr32("FnPtr", a, b);
+            var uses = new Identifier[] { b };
+            var defines = new Identifier[] { a };
+            var callStm = m.Call(fn, 4, uses, defines);
+            RunIndirectCallRewriter();
```

Comment:
Minor style issue: where possible, try grouping unit tests written in the [Arrange, Act, Assert](http://wiki.c2.com/?ArrangeActAssert) pattern so that the different parts stand out. Here, please surround `RunIndirectcallRewriter` with an empty line above and below. It makes it easier for an old guy like me to see the part where the code is under test.